opensearch_py_ml/ml_models/sentencetransformermodel.py

```diff
@@ -338,56 +328,39 @@ def read_queries(self, read_path: str, overwrite: bool = False) -> pd.DataFrame:
         df = df.drop_duplicates(subset=["query"])
         # for removing the "QRY:" token if they exist in passages
         df["passages"] = df.apply(lambda x: self.__qryrem(x), axis=1)
-        # shuffled data within dataframe
+        # shuffle data within dataframe
         df = df.sample(frac=1)
         return df
 
-    def load_sentence_transformer_example(
-        self, query_df, use_accelerate: bool = False
-    ) -> List[str]:
+    def load_sentence_transformer_example(self, query_df) -> List[List[str]]:
```

Comment:
I always felt this is misleading name. Can we change it to something like "load_training_data" ?